tests/__snapshots__/test_snapshots.yaml

```diff
@@ -36,12 +36,12 @@ test_set:
     this: null
 test_set.1:
   data: !!set
-    this: null
-    is: null
     a: null
     ? !!python/object/apply:builtins.frozenset
     - - nested, set
     : null
+    is: null
```

Comment:
yaml should be sorting the keys in alphabetical order, so having "tihs" and "is" after "a" is correct. Perhaps an issue with different pyaml versions, or a limitation of "set"?

Reply:
Might be a set limitation not having a specific order? Hmm, also if you merge master, I think I fixed the issue where it rewrites an existing passing snapshot. You can revert this snapshot change.

Reply:
Not sure why this is happening. Also thought I fixed the issue where it tries to rewrite existing passing snapshots

Reply:
I actually deleted this file and reran update snapshots